FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_mutual_auth/DemoTasks/MutualAuthMQTTExample.c

```diff
@@ -21,6 +21,8 @@
  *
  * http://www.FreeRTOS.org
  * http://aws.amazon.com/freertos
+ *
+ * 1 tab == 4 spaces!
```

Comment:
This is not required anymore. I think we will have to remove from other demos as well. The uncrustify.cfg converts all the tabs to spaces.